macosx/Controller.mm

```diff
@@ -3580,10 +3580,19 @@ - (void)checkAutoImportDirectory
             continue;
         }
 
+        NSDictionary<NSFileAttributeKey, id>* fileAttributes = [NSFileManager.defaultManager attributesOfItemAtPath:fullFile
+                                                                                                              error:nil];
+        if (fileAttributes.fileSize == 0)
```

Comment:
nit:
Is this some kind of dynamic lookup in NSDictionary?
I thought this will look something like this:
```objc
fileAttributes[NSFileSize]
```

Reply:
It's the same, just Apple's convenience way:
https://developer.apple.com/documentation/foundation/nsdictionary/1413465-filesize?language=objc

See https://developer.apple.com/documentation/foundation/nsfilemanager/1410452-attributesofitematpath#discussion :
>As a convenience, NSDictionary provides a set of methods (declared as a category on NSDictionary) for quickly and efficiently obtaining attribute information from the returned dictionary: [fileGroupOwnerAccountName](https://developer.apple.com/documentation/foundation/nsdictionary/1416788-filegroupowneraccountname), [fileModificationDate](https://developer.apple.com/documentation/foundation/nsdictionary/1408811-filemodificationdate), [fileOwnerAccountName](https://developer.apple.com/documentation/foundation/nsdictionary/1417533-fileowneraccountname), [filePosixPermissions](https://developer.apple.com/documentation/foundation/nsdictionary/1409446-fileposixpermissions), [fileSize](https://developer.apple.com/documentation/foundation/nsdictionary/1413465-filesize), [fileSystemFileNumber](https://developer.apple.com/documentation/foundation/nsdictionary/1408396-filesystemfilenumber), [fileSystemNumber](https://developer.apple.com/documentation/foundation/nsdictionary/1415329-filesystemnumber), and [fileType](https://developer.apple.com/documentation/foundation/nsdictionary/1416809-filetype).

Reply:
Thanks!